### ISSUE TITLE
Adding a command with --parent omits "Cmd" from the parent name in code

### ISSUE THREAD

**Issue by theothertomelliott** [spf13/cobra#514]
Using the CLI, if you create a parent command then add a child to it:

    $ cobra add parent
    $ cobra add child --parent parent

The generated code will not compile, as the child's init function will omit "Cmd" from the parent name.

    func init() {
	    parent.AddCommand(childCmd)

Is the expectation that --parent would be used with the variable name of the parent, so:

    $ cobra add child --parent parentCmd

Or should the same form be used for both parent and child names? If the former, a code change is required. If the latter, the help text could be expanded. Happy to submit a PR for either.

**Comment by bogem** [comment]
As in help example represented
``` 
-p, --parent string    name of parent command for this command (default "RootCmd")
```
you should use `Cmd` at the end of parent name.

**Comment by bogem** [comment]
PR for improving the help is welcome!

**Comment by bogem** [comment]
Fixed in #514
